src/Misc/XMLwrapper.cpp

```diff
@@ -528,39 +528,39 @@ void XMLwrapper::getparstr(const string &name, char *par, int maxstrlen) const
 
     if(tmp == NULL)
         return;
-    if(tmp->child == NULL)
+    if(mxmlGetFirstChild(tmp) == NULL)
         return;
-    if(tmp->child->type == MXML_OPAQUE) {
-        snprintf(par, maxstrlen, "%s", tmp->child->value.element.name);
+    if(mxmlGetType(mxmlGetFirstChild(tmp)) == MXML_OPAQUE) {
+        snprintf(par, maxstrlen, "%s", mxmlGetOpaque(mxmlGetFirstChild(tmp)));
         return;
     }
-    if((tmp->child->type == MXML_TEXT)
-       && (tmp->child->value.text.string != NULL)) {
-        snprintf(par, maxstrlen, "%s", tmp->child->value.text.string);
+    if((mxmlGetType(mxmlGetFirstChild(tmp)) == MXML_TEXT)
+       && (mxmlGetFirstChild(tmp) != NULL)) {
+        snprintf(par, maxstrlen, "%s", mxmlGetText(mxmlGetFirstChild(tmp),NULL));
         return;
     }
 }
 
 string XMLwrapper::getparstr(const string &name,
                              const std::string &defaultpar) const
 {
-    const mxml_node_t *tmp = mxmlFindElement(node,
+    mxml_node_t *tmp = mxmlFindElement(node,
                                              node,
```

Comment:
Changing this line results in the old alignment of other args (node, "string", "name", name.c_str()) being incorrect as they are now indented too far for the new code.